redash/tasks.py

```diff
@@ -2,6 +2,11 @@
 import logging
 from flask.ext.mail import Message
 import redis
+import hipchat
+import requests
+import json
+from bson import json_util
```

Comment:
This will require users to install PyMongo, which isn't a requirement for re:dash. Please use `json_dumps` from `redash.utils` instead of `json.dumps` with `json_util`.